plugins/destination/postgresql/cloud-config-ui/src/App.tsx

```diff
@@ -1,51 +1,65 @@
-import Stack from '@mui/material/Stack';
 import CssBaseline from '@mui/material/CssBaseline';
 import ThemeProvider from '@mui/material/styles/ThemeProvider';
 import createTheme from '@mui/material/styles/createTheme';
-import { Form } from './form';
 import { Fragment, useMemo } from 'react';
 import { createThemeOptions } from '@cloudquery/cloud-ui';
 import { pluginUiMessageHandler } from './utils/messageHandler';
-import { prepareInitialValues } from './utils/prepareInitialValues';
-import { CloudAppMock, useFormHeightChange, useFormInit } from '@cloudquery/plugin-config-ui-lib';
+import {
+  CloudAppMock,
+  ConfigUIForm,
+  // useFormHeightChange,
+  useFormInit,
+  PluginContextProvider,
+} from '@cloudquery/plugin-config-ui-lib';
+import { useConfig } from './hooks/useConfig';
 import Box from '@mui/material/Box';
-import { Guides } from './guides';
+import { prepareSubmitValues } from './utils/prepareSubmitValues';
 
 const useCloudAppMock =
   (process.env.REACT_APP_USE_CLOUD_APP_MOCK === 'true' || process.env.NODE_ENV !== 'production') &&
   window.self === window.top;
 const DevWrapper = useCloudAppMock ? CloudAppMock : Fragment;
 // eslint-disable-next-line unicorn/prefer-module
-const devWrapperProps: any = useCloudAppMock ? require('./.env.json') : undefined;
+const { plugin, ...devWrapperProps }: any = useCloudAppMock ? require('./.env.json') : {};
+
+const pluginProps = useCloudAppMock
+  ? plugin
+  : {
+      team: process.env.REACT_APP_PLUGIN_TEAM,
+      kind: process.env.REACT_APP_PLUGIN_KIND,
+      name: process.env.REACT_APP_PLUGIN_NAME,
+      version: process.env.REACT_APP_PLUGIN_VERSION,
+    };
 
 function App() {
-  const { initialValues, initialized, isManagedDestination } = useFormInit(
+  const { initialValues, initialized, teamName, context } = useFormInit(
     pluginUiMessageHandler,
     false,
   );
-  useFormHeightChange(pluginUiMessageHandler);
+  // useFormHeightChange(pluginUiMessageHandler);
 
   const theme = useMemo(() => createTheme(createThemeOptions()), []);
 
+  const config = useConfig({ initialValues });
```

Comment:
Previously we called `prepareInitialValues` before passing them to the Form to be rendered, maybe we also want to do this here - until we refactor the util functions.